mlflow/store/model_registry/sqlalchemy_store.py

```diff
@@ -206,7 +206,11 @@ def list_registered_models(self):
             return [sql_registered_model.to_mlflow_entity()
                     for sql_registered_model in session.query(SqlRegisteredModel).all()]
 
-    def search_registered_models(self, filter_string):
+    def search_registered_models(self,
+                                 filter_string=None,
+                                 max_results=None,
+                                 order_by=None,
```

Comment:
The merged backend implementation does not implement anything for `order_by`, so it's probably a good idea to either implement it, not include the parameter in the function for the store, or document it

Reply:
raised `NotImplementedError` now